### PR TITLE
log messages

### PR DESCRIPTION
**Pull request by DaanHoogland**
### Description

This PR fixes some log messages encountered during troubleshooting.

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (improves an existing feature and functionality)
- [x] Cleanup (Code refactoring and cleanup, that may add test cases)
- [ ] build/CI

#### Feature/Enhancement Scale

- [ ] Major
- [ ] Minor

#### Bug Severity

- [ ] BLOCKER
- [ ] Critical
- [ ] Major
- [ ] Minor
- [ ] Trivial

### CODE REVIEW

File: plugins/integrations/kubernetes-service/src/main/java/com/cloud/kubernetes/cluster/utils/KubernetesClusterUtil.java [hunk 1]
```diff
@@ -266,13 +266,11 @@ public static boolean isKubernetesClusterControlVmRunning(final KubernetesCluste
                 socket.connect(new InetSocketAddress(ipAddress, port), 10000);
                 controlVmRunning = true;
             } catch (IOException e) {
-                if (LOGGER.isInfoEnabled()) {
-                    LOGGER.info(String.format("Waiting for Kubernetes cluster : %s control node VMs to be accessible", kubernetesCluster.getName()));
-                }
+                LOGGER.info("Waiting for Kubernetes cluster : %s control node VMs to be accessible", kubernetesCluster.getName());
```
Comment (JoaoJandre) [inline plugins/integrations/kubernetes-service/src/main/java/com/cloud/kubernetes/cluster/utils/KubernetesClusterUtil.java:269]:
```suggestion
                LOGGER.info("Waiting for Kubernetes cluster : {} control node VMs to be accessible", kubernetesCluster.getName());
```

File: plugins/integrations/kubernetes-service/src/main/java/com/cloud/kubernetes/cluster/utils/KubernetesClusterUtil.java [hunk 2]
```diff
@@ -266,13 +266,11 @@ public static boolean isKubernetesClusterControlVmRunning(final KubernetesCluste
...
                 try {
                     Thread.sleep(10000);
                 } catch (InterruptedException ex) {
-                    LOGGER.warn(String.format("Error while waiting for Kubernetes cluster : %s control node VMs to be accessible", kubernetesCluster.getName()), ex);
+                    LOGGER.warn("Error while waiting for Kubernetes cluster : %s control node VMs to be accessible", kubernetesCluster.getName(), ex);
```
Comment (JoaoJandre) [inline plugins/integrations/kubernetes-service/src/main/java/com/cloud/kubernetes/cluster/utils/KubernetesClusterUtil.java:273]:
```suggestion
                    LOGGER.warn("Error while waiting for Kubernetes cluster : {} control node VMs to be accessible", kubernetesCluster.getName(), ex);
```

**Comment by DaanHoogland** [comment]
@blueorangutan package

**Comment by DaanHoogland** [comment]
I have seen https://github.com/apache/cloudstack/actions/runs/9187846058/job/25266387284?pr=9093#step:7:5498 in other PRs as well. I think we merged something that is not correct :(

**Review by shwstppr — COMMENTED** [review]
@DaanHoogland will this be an ongoing effort or is the scope of changes limited to currently changed log messages?

**Comment by DaanHoogland** [comment]
> @DaanHoogland will this be an ongoing effort or is the scope of changes limited to currently changed log messages?

log in general is an ongoing issue of course, but this PR is because of specific things I saw during a trouble shooting session. I do not plan any further work for now.

**Review by JoaoJandre — APPROVED** [review]
CLGTM

File: plugins/integrations/kubernetes-service/src/main/java/com/cloud/kubernetes/cluster/actionworkers/KubernetesClusterResourceModifierActionWorker.java [hunk 3]
```diff
@@ -263,9 +263,7 @@ protected DeployDestination plan(final long nodesCount, final DataCenter zone, f
                     logger.debug(String.format("Checking host : %s for capacity already reserved %d", h.getName(), reserved));
                 }
                 if (capacityManager.checkIfHostHasCapacity(h.getId(), cpu_requested * reserved, ram_requested * reserved, false, cpuOvercommitRatio, memoryOvercommitRatio, true)) {
-                    if (logger.isDebugEnabled()) {
-                        logger.debug(String.format("Found host : %s for with enough capacity, CPU=%d RAM=%s", h.getName(), cpu_requested * reserved, toHumanReadableSize(ram_requested * reserved)));
-                    }
+                    logger.debug(String.format("Found host {} with enough capacity: CPU={} RAM={}", h.getName(), cpu_requested * reserved, toHumanReadableSize(ram_requested * reserved)));
```
Comment (winterhazel) [inline plugins/integrations/kubernetes-service/src/main/java/com/cloud/kubernetes/cluster/actionworkers/KubernetesClusterResourceModifierActionWorker.java:266]:
```suggestion
                    logger.debug("Found host {} with enough capacity: CPU={} RAM={}", h.getName(), cpu_requested * reserved, toHumanReadableSize(ram_requested * reserved));
```

**Review by winterhazel — CHANGES_REQUESTED** [review]

**Comment by DaanHoogland** [comment]
@blueorangutan package

**Comment by DaanHoogland** [comment]
@blueorangutan test alma9 vmware-70u3

**Comment by DaanHoogland** [comment]
@blueorangutan package

**Review by winterhazel — APPROVED** [review]
CLGTM, did not test.

**Comment by DaanHoogland** [comment]
@blueorangutan test alma9 vmware-70u3

**Pull request merged**